Editor/AGS.CScript.Compiler/Preprocessor.cs

```diff
@@ -356,6 +356,7 @@ private string RemoveComments(string text)
 						int endOfString = FindIndexOfMatchingCharacter(text, i, text[i]);
 						if (endOfString < 0)
 						{
+							RecordError(ErrorCode.UserDefinedError, "Unterminated string");

```

Comment:
UserDefinedError is an incorrect error type for this; it's for errors "created" by scripters with #error command, not actual mistakes made by user.